### PR TITLE
Adapt package scripts to allow installing two different versions in parallel

### PR DESCRIPTION
**Pull request by david-eckardt-sociomantic**
For `make pkg` the `PKG_SUFFIX` environment variable can now be used to specify a suffix to the package name.

### CODE REVIEW

**Comment by david-eckardt-sociomantic** [comment]
Damn, makd v2.1.0 doesn't work with any ocean release:
`./submodules/ocean/src/ocean/core/UnitTestRunner.d(932): Error: only one main allowed`.

@nemanja-boric-sociomantic, we need to adapt this to makd v1.11.1.

**Comment by leandro-lucarella-sociomantic** [comment]
Use v2.1.2! v2.x.0 releases were broken

**Comment by leandro-lucarella-sociomantic** [comment]
Although I think that was not the error :thinking:

**Comment by david-eckardt-sociomantic** [comment]
No, the error is the same with makd v2.1.2

**Comment by leandro-lucarella-sociomantic** [comment]
Are you sure you followed the [migration instructions](https://github.com/sociomantic-tsunami/makd/releases/tag/v2.0.0) (in particular the ones about versioning out the `main()`s)?

**Comment by Burgos** [comment]
Yeah, that's not done, so I think that's the problem. Also, `test` needs to be renamed to `integrationtest`.

**Comment by Burgos** [comment]
(The main cause of the issue is that we tested only making the package, but never run `make test` locally).

**Comment by david-eckardt-sociomantic** [comment]
Updated, using makd v2.1.2 and turtle v8.2.1 and properly adapting to the new makd version.

File: pkg/dmqnode-common.pkg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import defaults
```
Comment (leandro-lucarella-sociomantic) [inline pkg/dmqnode-common.pkg]: It looks like this might be in the wrong commit.

Comment (david-eckardt-sociomantic) [reply pkg/dmqnode-common.pkg]: Updated, it should be in the right commit now.

**Comment by gavin-norman-sociomantic** [comment]
Leaving for @nemanja-boric-sociomantic to merge.

**Comment by nemanja-boric-sociomantic** [comment]
Roger that.

**Pull request merged**